frontend/src/modules/fluentoriginal/components/FluentOriginalString.js

```diff
@@ -8,15 +8,19 @@ import RichString from './RichString';
 import SimpleString from './SimpleString';
 import SourceString from './SourceString';
 
+import ContextIssueButton from 'modules/entitydetails/components/ContextIssueButton';
```

Comment:
"Outside of the module, always import from the module's index, and never from specific files. This allows us to easily evolve modules and keep things decoupled, which reduces code complexity."

https://github.com/mozilla/pontoon/tree/master/frontend#modules